src/types/to_sql.rs

```diff
@@ -224,6 +224,13 @@ impl ToSql for Vec<u8> {
     }
 }
 
+impl<const N: usize> ToSql for [u8; N] {
```

Comment:
Huh, we have a `ToSql for [u8]`, and I have expected the `&[u8; N]` to unsize into `&[u8]` and use that one. I guess I can see why the cases in your test might be too hard for the compiler to figure out.